src/libponyrt/CMakeLists.txt

```diff
@@ -127,12 +127,15 @@ if(PONY_RUNTIME_BITCODE)
         message(FATAL_ERROR "You can only use runtime-bitcode with Clang.")
     endif()
 
+    set(_c_flags ${CMAKE_C_FLAGS})
+    separate_arguments(_c_flags)
+
     set(_ll_src_all ${_c_src})
     foreach(_src_file ${_ll_src_all})
         #message("${libponyrt_SOURCE_DIR}/${_src_file} -> ${libponyrt_BINARY_DIR}/${_src_file}.bc")
         get_filename_component(_src_dir ${_src_file} DIRECTORY)
         add_custom_command(
-            COMMAND mkdir -p "${libponyrt_BINARY_DIR}/${_src_dir}" && ${CMAKE_C_COMPILER} $<IF:$<BOOL:${CMAKE_C_COMPILER_TARGET}>,--target=${CMAKE_C_COMPILER_TARGET},> -DBUILD_COMPILER="${CMAKE_C_COMPILER_VERSION}" -D_FILE_OFFSET_BITS=64 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -DLLVM_BUILD_MODE=${PONY_LLVM_BUILD_MODE} -DLLVM_VERSION="${LLVM_VERSION}" -DPONY_ALWAYS_ASSERT -DPONY_COMPILER="${CMAKE_C_COMPILER}" -DPONY_ARCH="${CMAKE_SYSTEM_PROCESSOR}" -DPONY_BUILD_CONFIG="release" -DPONY_USE_BIGINT -DPONY_VERSION="${PONYC_VERSION}" -DPONY_VERSION_STR="${PONYC_VERSION} [release]\\nCompiled with: LLVM ${LLVM_VERSION} -- ${CMAKE_C_COMPILER_ID}-${CMAKE_C_COMPILER_VERSION}-${CMAKE_C_COMPILER_ARCHITECTURE_ID}" -fexceptions -std=gnu11 -Werror -Wconversion -Wno-sign-conversion -Wno-atomic-alignment -Wextra -Wall -I. -I../common -emit-llvm -o "${libponyrt_BINARY_DIR}/${_src_file}.bc" -c ${_src_file}
```

Comment:
What does -std=gnu11 do?

Reply:
According to [this doc](https://clang.llvm.org/docs/LanguageExtensions.html#c11), I believe that it is a way to explicitly state that you want the C11 features outlined there (including some ones we use, like C11 atomics, alignas, etc). However, the doc also says that clang has those features always on because they are backwards compatible.

So I assume that `-std=gnu11` would only be a necessary arg if we were using gcc - which we never are here because only clang can generate bitcode.